### PR TITLE
feat(files): add Pale Oak Wood, Logs & Planks to Crafting Tweaks

### PR DESCRIPTION
**Pull request by SquatchHunter**
- add pale_oak_* recipes
- flatten any non flattened entries
- removed extra entries (in Powder to Glass)
- updated all instances of 1.21.x to 1.21.50

By checking the following boxes with an X, you ensure that:

- [x] The pack was tested ingame in at least one device.
- [x] The pack is an existing BT pack, is a missing pack from VT or is an accepted pack/change in a discussion.
- [x] The pack code follows the style guide.
- [x] The commits follow the contribution guidelines.
- [x] The PR follows the contribution guidelines.

- [x] (Optional) Tested in Windows
- [ ] (Optional) Tested in Android
- [ ] (Optional) Tested in iOS
- [ ] (Optional) Tested in any console
- [ ] (Optional) Tested in BDS

### CODE REVIEW

File: crafting_tweaks/files/quality_of_life/back_to_blocks/recipes/back_to_blocks/slabs/smooth_sandstone.recipe.json [hunk 1]
```diff
@@ -14,19 +14,16 @@
 		],
 		"key": {
 			"#": {
-				"item": "minecraft:stone_block_slab2",
-				"data": 6
+				"item": "minecraft:smooth_sandstone"
```
Comment (PAPISOP) [inline crafting_tweaks/files/quality_of_life/back_to_blocks/recipes/back_to_blocks/slabs/smooth_sandstone.recipe.json:17]: Should be `"minecraft:smooth_sandstone_slab"` not `"minecraft:smooth_sandstone"` lol

File: crafting_tweaks/files/quality_of_life/back_to_blocks/recipes/back_to_blocks/slabs/smooth_sandstone.recipe.json [hunk 2]
```diff
@@ -14,19 +14,16 @@
...
 			}
 		},
 		"unlock": [
 			{
-				"item": "minecraft:stone_block_slab2",
-				"data": 6
+				"item": "minecraft:smooth_sandstone"
```
Comment (PAPISOP) [inline crafting_tweaks/files/quality_of_life/back_to_blocks/recipes/back_to_blocks/slabs/smooth_sandstone.recipe.json:22]: Should be `"minecraft:smooth_sandstone_slab"` not `"minecraft:smooth_sandstone"` lol

File: crafting_tweaks/files/quality_of_life/double_slabs/recipes/double_slabs/polished_diorite_slab.recipe.json [hunk 3]
```diff
@@ -19,8 +19,7 @@
 			}
 		],
 		"result": {
-			"item": "minecraft:stone_block_slab3",
-			"data": 5,
+			"item": "minecraft:polished_andesite_slab",
```
Comment (PAPISOP) [inline crafting_tweaks/files/quality_of_life/double_slabs/recipes/double_slabs/polished_diorite_slab.recipe.json:22]: Should be `"minecraft:polished_diorite_slab"` not `"minecraft:polished_andesite_slab"`

File: crafting_tweaks/files/quality_of_life/powder_to_glass/recipes/powder_to_glass/black_stained_glass.recipe.json
Comment (PAPISOP) [inline crafting_tweaks/files/quality_of_life/powder_to_glass/recipes/powder_to_glass/black_stained_glass.recipe.json:1]: The formatting is not done in powder_to_glass pack and can you explain why you removed the `"group"` field from each recipe from this pack?

File: crafting_tweaks/files/quality_of_life/sandstone_dyeing/recipes/sandstone_dyeing/smooth_red_sandstone.recipe.json [hunk 4]
```diff
@@ -11,12 +11,10 @@
 		"priority": -1,
 		"ingredients": [
 			{
-				"item": "minecraft:sandstone",
-				"data": 3
+				"item": "minecraft:red_sandstone"
```
Comment (PAPISOP) [inline crafting_tweaks/files/quality_of_life/sandstone_dyeing/recipes/sandstone_dyeing/smooth_red_sandstone.recipe.json:14]: Should be `"minecraft:smooth_sandstone"` not `"minecraft:red_sandstone"`

File: crafting_tweaks/files/quality_of_life/sandstone_dyeing/recipes/sandstone_dyeing/smooth_red_sandstone.recipe.json [hunk 5]
```diff
@@ -11,12 +11,10 @@
...
 			},
 			{
-				"item": "minecraft:sandstone",
-				"data": 3
+				"item": "minecraft:red_sandstone"
```
Comment (PAPISOP) [inline crafting_tweaks/files/quality_of_life/sandstone_dyeing/recipes/sandstone_dyeing/smooth_red_sandstone.recipe.json:17]: Should be `"minecraft:smooth_sandstone"` not `"minecraft:red_sandstone"`

**Review by PAPISOP — CHANGES_REQUESTED** [review]
1. Remove the `"data"` field from `files\more_blocks\more_stairs\recipes\more_stairs\quartz_stairs.recipe.json`, `files\quality_of_life\alternate_cobblestone\recipes\alternate_cobblestones\dispenser.recipe.json`, `files\quality_of_life\alternate_cobblestone\recipes\alternate_cobblestones\dropper.recipe.json`, `files\quality_of_life\alternate_cobblestone\recipes\alternate_cobblestones\piston.recipe.json`, `files\quality_of_life\back_to_blocks\recipes\back_to_blocks\slabs\end_bricks.recipe.json`, `files\quality_of_life\back_to_blocks\recipes\back_to_blocks\slabs\quartz_block.recipe.json` and `files\quality_of_life\double_slabs\recipes\double_slabs\red_sandstone_slab.recipe.json`
2. Format the `powder_to_glass` pack
3. Make sure the `minecraft:` namespace is added to `piston` in the output in `files\quality_of_life\alternate_cobblestone\recipes\alternate_cobblestones\piston.recipe.json`

File: crafting_tweaks/files/more_blocks/more_bark/recipes/more_bark/stripped_wood/stripped_pale_oak_wood.json
[the remark this answers is not in the document]
Comment (SquatchHunter) [reply crafting_tweaks/files/more_blocks/more_bark/recipes/more_bark/stripped_wood/stripped_pale_oak_wood.json:1]: I think this must of somehow pushed an old commit cause the naming and all those data fields were removed

**Comment by SquatchHunter** [comment]
closing in favour of splitting into two new PRs
will edit comment when I have PR numbers, likely #431 & #432

File: crafting_tweaks/files/quality_of_life/sandstone_dyeing/recipes/sandstone_dyeing/smooth_red_sandstone.recipe.json (hunk 4 above)
Comment (SquatchHunter) [reply crafting_tweaks/files/quality_of_life/sandstone_dyeing/recipes/sandstone_dyeing/smooth_red_sandstone.recipe.json:14]: actually 🤓👆
it should be smooth_red_sandstone
we are both wrong XD

File: crafting_tweaks/files/quality_of_life/sandstone_dyeing/recipes/sandstone_dyeing/smooth_red_sandstone.recipe.json (hunk 5 above)
Comment (SquatchHunter) [reply crafting_tweaks/files/quality_of_life/sandstone_dyeing/recipes/sandstone_dyeing/smooth_red_sandstone.recipe.json:17]: actually 🤓👆
it should be smooth_red_sandstone
we are both wrong XD

Comment (PAPISOP) [reply crafting_tweaks/files/quality_of_life/sandstone_dyeing/recipes/sandstone_dyeing/smooth_red_sandstone.recipe.json:17]: No? it's smooth sandstone which is used to convert it to smooth red sandstone 😂

Comment (SquatchHunter) [reply crafting_tweaks/files/quality_of_life/sandstone_dyeing/recipes/sandstone_dyeing/smooth_red_sandstone.recipe.json:17]: ah yea its in sandstone dying
which isnt a pack I editted on purpose, minus the change to the version number, a random find + replace must of foobar'd this

**Pull request closed without merging**